spec/features/reviews/edit_spec.rb

```diff
@@ -53,5 +53,18 @@
       end
     end
 
+    it "cannot edit a review with a field left blank" do
+      visit "/shelters/#{@shelter_1.id}/reviews/#{@review_1.id}/edit"
+
+      fill_in 'Title', with: 'An Ok Shelter'
+      fill_in 'Rating', with: 3
+      fill_in 'Content', with: ''
```

Comment:
Took me a minute to realize I had to fill in the field with a blank because it was auto-filled with the previous info.